pageserver/src/tenant/timeline.rs

```diff
@@ -293,6 +293,10 @@ pub struct Timeline {
     /// Completion shared between all timelines loaded during startup; used to delay heavier
     /// background tasks until some logical sizes have been calculated.
     initial_logical_size_attempt: Mutex<Option<completion::Completion>>,
+
+    /// Load or creation time information about the disk_consistent_lsn and when the loading
+    /// happened. Used for consumption metrics.
+    pub(crate) loaded_as: (Lsn, SystemTime),
```

Comment:
No actually it's not, because it will of course update, so I am thinking this will be the best possible estimate as for a "closest to the last value we sent over".